db/migrations/2025_03_21_112110_create_project_and_study_user_join_tables.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sql
+
+
+def up(conn):
+    query = """
+        CREATE TABLE ProjectUsers (
+            id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+
+            projectUniqueID VARCHAR(100) COLLATE utf8mb4_bin,
+            userUniqueID    VARCHAR(100) COLLATE utf8mb4_bin,
+
+            createdAt DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+            CONSTRAINT projectUniqueID FOREIGN KEY (projectUniqueID)
+                REFERENCES Project (projectUniqueID)
+                ON DELETE CASCADE ON UPDATE CASCADE
+        )
+    """
+    conn.execute(sql.text(query))
+
+    query = """
+        CREATE TABLE StudyUsers (
+            id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+
+            studyUniqueID VARCHAR(100) COLLATE utf8mb4_bin,
+            userUniqueID  VARCHAR(100) COLLATE utf8mb4_bin,
+
+            createdAt DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+            CONSTRAINT studyUniqueID FOREIGN KEY (studyUniqueID)
+                REFERENCES Study (studyUniqueID)
+                ON DELETE CASCADE ON UPDATE CASCADE
+        )
+    """
+    conn.execute(sql.text(query))
```

Comment:
These tables connect studies and projects with users that have the ability to update them.